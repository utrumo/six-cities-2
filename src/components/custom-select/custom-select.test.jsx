import React from 'react';
import renderer from 'react-test-renderer';
import CustomSelect from './custom-select.jsx';

it(`Should renders correctly`, () => {
  const tree = renderer
    .create(<CustomSelect
      label="testLable"
      options={[`first`, `second`]}
      onChange={jest.fn()}
      value="second"
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
