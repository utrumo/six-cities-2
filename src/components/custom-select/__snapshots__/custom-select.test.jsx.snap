// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders correctly 1`] = `
<label>
  <span
    className="places__sorting-caption"
  >
    testLable
  </span>
   
  <span
    className="places__sorting-type"
    onClick={[Function]}
    onKeyDown={[Function]}
    tabIndex="0"
  >
    second
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom"
  >
    <li
      className="places__option"
      onClick={[Function]}
    >
      first
    </li>
    <li
      className="places__option places__option--active"
      onClick={[Function]}
    >
      second
    </li>
  </ul>
</label>
`;
